sycl/test-e2e/ESIMD/unified_memory_api/Inputs/scatter.hpp

```diff
@@ -489,8 +489,7 @@ bool testLACC(queue Q, uint32_t MaskStride,
          if (LocalID == 0) {
            for (int I = 0; I < Threads * N; I++) {
              simd<T, 1> InVec(Out + GlobalElemOffset + I);
-             simd<uint32_t, 1> Offsets(I * sizeof(T), sizeof(T));
-             scatter<T>(LocalAcc, Offsets, InVec);
+             LocalAcc[I] = InVec[0];
```

Comment:
nit: why do you even need simd<T,1> if you use such assignment? Why not to have then this:
```suggestion
             LocalAcc[I] = Out[GlobalElemOffset + I];
```